wpilibj/src/athena/java/edu/wpi/first/wpilibj/SpeedControllerGroup.java

```diff
@@ -0,0 +1,111 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016-2017. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Allows multiple {@link SpeedController} objects to be linked together.
+ */
+public class SpeedControllerGroup<T extends SpeedController> implements SpeedController {
+
+  private boolean m_isInverted = false;
+  private final List<T> m_speedControllers;
+
+  /**
+   * Create a new SpeedControllerGroup with no SpeedControllers.
+   */
+  public SpeedControllerGroup() {
+    m_speedControllers = new ArrayList<>();
+  }
+
+  /**
+   * Create a new SpeedControllerGroup with the provided SpeedControllers.
+   *
+   * @param speedControllers The SpeedControllers to add
+   */
+  public SpeedControllerGroup(Collection<T> speedControllers) {
+    m_speedControllers = new ArrayList<>(speedControllers);
+  }
+
+  /**
+   * Adds the provided SpeedController to the SpeedControllerGroup.
+   *
+   * @param speedController The SpeedController to add
+   */
+  public void add(T speedController) {
```

Comment:
Why is this class mutable? Nobody is going to be creating new speed controller objects after initialization.